__tests__/app.test.js

```diff
@@ -331,4 +330,43 @@ describe('app.js', () => {
         });
     });
   });
+  describe('10. GET /api/reviews (queries)', () => {
+    test('returns an array of objects where each object category is equal to the query', () => {
+      return request(app)
+        .get('/api/reviews?category=social deduction')
+        .expect(200)
+        .then((response) => {
+          expect(Array.isArray(response.body.reviews)).toBe(true);
+          expect(response.body.reviews.length).toBe(11);
+          expect(
+            response.body.reviews.forEach((review) => {
+              expect(typeof review).toBe('object');
+              expect(review).toEqual(
+                expect.objectContaining({
+                  owner: expect.any(String),
+                  designer: expect.any(String),
+                  category: expect.toBeString('social deduction'),
+                })
+              );
+            })
+          );
+        });
+    });
+    test('returns a 404 error when category is not found', () => {
+      return request(app)
+        .get('/api/reviews?category=rpg')
+        .expect(404)
+        .then((response) => {
+          expect(response.body.message).toBe('No matching results found');
+        });
+    });
+    test('returns a 400 error when trying to sort by invalid column', () => {
+      return request(app)
+        .get('/api/reviews?sort_by=banana')
+        .expect(400)
+        .then((response) => {
+          expect(response.body.message).toBe('Invalid sorting query');
+        });
+    });
+  });
```

Comment:
there should be some more tests here for the queries section - think about the orders desc and asc, other sort_by queries rather than just a category to really test that your requests are working as they should

Reply:
![Screenshot from 2023-01-18 12-42-42](https://user-images.githubusercontent.com/115625241/213425085-35185a3d-c642-4b7d-ad81-fc9f05f7b68a.png)

Reply:
![Screenshot from 2023-01-19 10-59-02](https://user-images.githubusercontent.com/115625241/213425248-061da941-7f70-4cb4-b4bf-f93bba220941.png)